src/tests/ftest/server/metadata.py

```diff
@@ -184,6 +183,60 @@ def destroy_all_containers(self):
         self.container = []
         return len(errors) == 0
 
+    def destroy_num_containers(self, num):
+        """Destroy some of the created containers.
+
+        Args:
+            num (int): Number of containers to destroy
+
+        Returns:
+            bool: True if all of the containers were destroyed successfully;
+                False otherwise
+
+        """
+        if num > len(self.container):
+            num = len(self.container)
+        self.log.info("Destroying %d containers", num)
+        errors = self.destroy_containers(self.container[0:(num - 1)])
```

Comment:
Sorry I didn't notice this before. This is actually off by one. The upper index is exclusive. For example
```
>>> my_list = [0,1,2,3]
>>> my_list[0:0]
[]
>>> my_list[0:1]
[0]
```
So you would want `[0:num]`